src/leveled_codec.erl

```diff
@@ -112,10 +112,17 @@
         {index_specs(), infinity|integer()}. % {KeyChanges, TTL}
 -type maybe_lookup() ::
         lookup|no_lookup.
+-type actual_regex() ::
+        {re_pattern, term(), term(), term(), term()}.
```

Comment:
would it not be nicer to use `re:mp()` here, to hide the details?

Reply:
in leveled_runner you replaced it by `any()`, but I prefer here to wait until OTP27 is lowest supported and replace it then.